dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.35/V2_35_11__Add_new_Map_View_Event_Status_column.sql

```diff
@@ -0,0 +1,4 @@
+
+-- Add a new column "eventstatus" to the mapview table.
+
+ALTER TABLE mapview ADD COLUMN eventstatus VARCHAR(50);
```

Comment:
Change to "add column if not exists".

Reply:
Yes, good one. Will push in a few minutes.

Reply:
Done